CONTRIBUTING.adoc

```diff
@@ -0,0 +1,308 @@
+2. Create a new branch from your `main` branch and perform your work on this new branch, avoiding direct changes to the `main` branch.
+3. Once your work is completed and committed to your branch, push the changes to your forked repository.
+4. link:https://help.github.com/articles/creating-a-pull-request-from-a-fork/[Create a pull request (PR)] from your branch to the `main` branch of this repository.
+
+Forking only needs to be done once. After that, you can keep your fork up to date with the upstream repository thanks to *Sync fork*.
+You can make further changes to your forked repository using the GitHub website file editor or by working on a local clone, as described below.
+
+[[newcomers]]
+=== Information for newcomers
+
+The documentation below describes common cases for contributors.
+If you have any questions, please do not hesitate to ask in link:https://app.gitter.im/\#/room/#jenkinsci_newcomer-contributors:gitter.im[newcomers] or link:https://app.gitter.im/\#/room/#jenkins/docs:matrix.org[Documentation SIG] Gitter chats.
+If you have any feedback about your contributor experience, please also share it with us in these channels so that we can improve our guidelines.
+
+You can also find some newbie-friendly issues in our task tracker:
+
+* link:https://github.com/jenkins-infra/docs.jenkins.io/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22[Good First Issues on GitHub]
+
+If you found an issue that you would like to work on, you are welcome to go ahead and start contributing.
+Please don't ask to be assigned to an issue, we don't assign individuals to issues, just go ahead and start working on it.
+If you want to avoid having other people work on the same issue, feel free to add a comment to the issue stating that you're working on it.
+If the last comment on the issue is older than a week and there is no PR linked to the issue, you can assume nobody is working on it.
+If you need any help, please ask in the xref:contacts[].
+
+[[contacts]]
+=== Communication channels
+
+The Jenkins website is largely maintained by the link:https://jenkins.io/sigs/docs/[Jenkins documentation] special interest group.
+This group has various communication channels which can be used to discuss contributing to the website:
+
+* link:https://app.gitter.im/#/room/#jenkins/docs:matrix.org[Gitter chat]
+* link:https://jenkins.io/sigs/docs/#meetings[Regular meetings]
+
+== Structure of docs.jenkins.io
+
+The `docs.jenkins.io` repository contains two distinct parts:
+
+1. **Antora** (Versioned)
+   - link: link:/docs[/docs]
+   - This part of the site is built using Antora, a static site generator designed for documentation. It is version-controlled, meaning that different versions of Jenkins documentation are maintained.
+   - You can find more information in the link:https://github.com/jenkins-infra/docs.jenkins.io/tree/main/docs#readme[Antora README.md].
+
+2. **Gatsby** (Non-Versioned)
+   - link: link:/site[/site]

```

Comment:
```suggestion
   - link: link:./site/README.md[site]
```

Reply:
Same here, the link to the readme is 2 rows below

Reply:
Same here, your link is broken and we don't want a slash in the text. I have just updated the suggestion. Please commit the change.